radboudumc-theme/sass/_typography.scss

```diff
@@ -12,17 +12,39 @@ body {
   // No web fonts are used so maximize client rendering of the system font
   text-rendering: geometricPrecision;
   -webkit-font-smoothing: antialiased;
+
+  color: rgb(50, 50, 50);
```

Comment:
Text color can be changed globally by adding a new variable in _variables.scss (override the default variable used by Bootstrap). See https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss